<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>debug_zuhe</class>
 <widget class="QWidget" name="debug_zuhe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/image/image/plc/comTest.png);</string>
  </property>
  <widget class="QPushButton" name="pbExit">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>420</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbStopAll">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>420</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>全部停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCybjy">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>采样泵进样</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSydjcc">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>水样到检测池？</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbJccdsy">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>检测池到水样？</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDlgdjcc">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>定量管到检测池？</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbZlhdsy">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>载流环到水样</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbKqdxg">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>空气到吸管</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbZlhddlg">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>载流环到定量管</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbXdydjcc">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>消毒液到检测池？</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbWjsdjcc">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>无菌水到检测池？</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbWjsdzlh">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>无菌水到载流环？</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbXgddlg">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>吸管到定量管</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbJccdfy">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>检测池？到废液</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbXdydzlh">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>消毒液到载流环</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCy">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>采样</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbGlqx">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>管路清洗</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbJccqx">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>检测池清洗</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbQbqx">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>全部清洗</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbZdcsw">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>60</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>机械臂到初始位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbZdxqw">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>110</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>机械臂到吸取位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbZdpyw">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>160</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>机械臂到排液位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbZpdpyw">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>260</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>转盘到排液位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbZpdpw">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>310</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>转盘到瓶位？</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_TestID">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>368</y>
     <width>45</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_BottomID">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>368</y>
     <width>45</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_PlayTime">
   <property name="geometry">
    <rect>
     <x>383</x>
     <y>368</y>
     <width>65</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Max_Time">
   <property name="geometry">
    <rect>
     <x>623</x>
     <y>368</y>
     <width>65</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>300</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_onoff">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>265</y>
     <width>25</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>关</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_setup">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>265</y>
     <width>30</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
